ci/main.sh

```diff
@@ -5,6 +5,7 @@ set -eux
 
 : "${RNP_TESTS:=.*}"
 : "${LD_LIBRARY_PATH:=}"
+export RNP_OVERRIDE_BRANCH=master
```

Comment:
Should we set this here or in the job definition?

Reply:
We could go either way, I'm thinking it's maybe easier here since it's one less env var for a developer have to set if they are running the CI scripts locally (which is sometimes useful if you're too impatient for the 2-core CI systems).

Reply:
Got it. Thanks 👍